WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -181,6 +193,11 @@ class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	 * @return array
 	 */
 	public function register() {
+		// Get a list of all PHP native build in functions.
```

Comment:
Remove "build in" (should be "built-in" anyway), since "native" already means that.

```
// Get a list of all PHP native functions.
```

Reply:
Fixed up (including in the property docblock).